src/rum/telemetry.h

```diff
@@ -0,0 +1,39 @@
+#include <datadog/telemetry/metrics.h>
+
+#include <format>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "version.h"
+
+namespace datadog {
+namespace nginx {
+namespace rum {
+namespace telemetry {
+
+template <typename... T>
+auto build_telemetry_tags(T&&... specific_tags) {
```

Comment:
I noticed that the term 'telemetry' appears both in the name and the namespace, which might be redundant. Would you consider renaming it to `build_tags` to improve clarity?